client/src/push/manager.js

```diff
@@ -21,7 +21,10 @@ class PushManager {
   async init() {
     const promises = [];
 
-    this.services.dummy = await initDummy(this);
+    const dummy = await initDummy(this);
```

Comment:
I'm not convinced this fixes it because I don't see how the thing that we call can ever return null. I think we should do more investigation.

Reply:
well let me clarify. It probably does fix the bug as we see it but it's masking something else. This same situation can probably happen for the other providers.

Reply:
Yea its protecting the problem not fixing it.